src/main/resources/org/jenkinsci/plugins/workflow/steps/ErrorStep/help.html

```diff
@@ -4,3 +4,9 @@
     You can also just <code>throw new Exception()</code>,
     but this step will avoid printing a stack trace.
 </div>
+<br>
+<p>The <a href="https://www.jenkins.io/redirect/pipeline-snippet-generator">Pipeline Snippet Generator</a> generates this example:
```

Comment:
Do we need thus long sentence? Or just "Example"?

Reply:
This is how they added an example in the git plugin, So I thought I had to do the same 
![image](https://user-images.githubusercontent.com/37863089/116005505-89b11d80-a607-11eb-953f-eb4cb6c96e3d.png)

Reply:
Ha!  Writers discuss this often.  The text @ciradu2204 wrote really is better English style but just saying "Example" or "for example" is cleaner and perhaps more accessible for many people.  Personally, I see merit in both arguments and go back and forth...  ;-)

Reply:
@stackScribe, so I should only say Example?

Reply:
Either form is acceptable but I would go with Oleg's suggestion.  I do like terser prose for a reference document but some writers disagree ;-)

Reply:
Thank you, let me correct that